src/SFA.DAS.AANHub.Application/Employers/Commands/CreateEmployerMember/CreateEmployerMemberCommandValidator.cs

```diff
@@ -1,31 +1,46 @@
 ﻿using FluentValidation;
 using SFA.DAS.AANHub.Application.Common;
 using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using static SFA.DAS.AANHub.Domain.Common.Constants;
 
 namespace SFA.DAS.AANHub.Application.Employers.Commands.CreateEmployerMember;
 
 public class CreateEmployerMemberCommandValidator : AbstractValidator<CreateEmployerMemberCommand>
 {
     private const string UserRefAlreadyCreatedErrorMessage = "UserRef already exists";
+    private const string ProfileValuesMustNotBeEmptyErrorMessage = "ProfileValues cannot be empty";
+    private const string InvalidProfileIdsErrorMessage = "Some of the profile ids are invalid for Employer user type";
 
     private readonly IEmployersReadRepository _employersReadRepository;
+    private readonly IProfilesReadRepository _profilesReadRepository;
 
-    public CreateEmployerMemberCommandValidator(IEmployersReadRepository employersReadRepository)
+    public CreateEmployerMemberCommandValidator(IEmployersReadRepository employersReadRepository, IProfilesReadRepository profilesReadRepository)
     {
         _employersReadRepository = employersReadRepository;
+        _profilesReadRepository = profilesReadRepository;
 
         Include(new CreateMemberCommandBaseValidator());
+
         RuleFor(c => c.UserRef)
             .NotEmpty()
             .MustAsync(async (command, _, _) => await IsNewUser(command.UserRef))
             .WithMessage(UserRefAlreadyCreatedErrorMessage);
 
-        RuleFor(c => c.Organisation)
-            .NotEmpty()
-            .MaximumLength(250);
-
         RuleFor(c => c.AccountId)
             .NotEmpty();
+
+        RuleFor(c => c.ProfileValues)
+            .NotEmpty()
+            .WithMessage(ProfileValuesMustNotBeEmptyErrorMessage)
+            .ForEach(x => x.SetValidator(new ProfileValueValidator()))
+            .MustAsync(async (profileValues, _) =>
```

Comment:
There is no coverage for this validation, empty and invalid ids both needs to be covered with positive tests.

Reply:
I think this one is covered as I can see the errors populating in testResult